### ISSUE TITLE
Cannot specify output chunk names

### ISSUE THREAD

**Issue by sanchitgn** [vuejs/vue-cli#1167]
### Version
3.0.0-beta.6

### Reproduction link
[https://codesandbox.io/s/oxm4l6857y](https://codesandbox.io/s/oxm4l6857y)

### Steps to reproduce
1. Specify a chunk using dynamic import syntax 
2. Add a chunk name `import(/* webpackChunkName: "profile" */ "./views/Profile/Profile.vue")`
3. Build `yarn run build`

### What is expected?
```  
   File                                      Size             Gzipped

  dist/js/profile.e9349658.js               31.66 kb         5.56 kb
```

### What is actually happening?
``` 
 File                                      Size             Gzipped

  dist/js/0.e9349658.js                     31.66 kb         5.56 kb
```

---
Sorry if this is has already been posted. Could not find a similar issue.
The other webpack comments (`webpackMode`) work fine

Possible duplicate [#1097](https://github.com/vuejs/vue-cli/issues/1097)

**Comment by LinusBorg** [comment]
You probably have to set `"comments": true` in the Babel config.

**Comment by sanchitgn** [comment]
Setting `"comments": true` in `.babelrc` doesn't seem to work

**Comment by LinusBorg** [comment]
...which is why I alredy removed my comment. I misread your OP.

**Comment by sanchitgn** [comment]
@LinusBorg Check [this repo](https://github.com/sanchitgn/chunk). After specifying `webpackChunkName` in [router.js](https://github.com/sanchitgn/chunk/blob/master/src/router.js#L12:53), the output is still `0.[hash].js`. The project was created with the latest vue-cli. The only change it the dynamic `Home` component

**Comment by LinusBorg** [comment]
I think this line is the problematic one:

https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-service/lib/config/prod.js#L8

Please try this in vue.config.js (can't do that myelf right now):
```javascript
module.exports {
  chainWebpack: config => {
    config.output.chunkFilename(`js/[name].[chunkhash:8].js`)

  }
}
```

If that solves the problem, we should change ths with a PR.

**Comment by sanchitgn** [comment]
Yup, that solves it. It makes sense to add [name] in the chunk file name

**Comment by zigomir** [comment]
Looks like this one can be closed.

**Comment by doublemarked** [comment]
This doesn't work if you're building as `lib` or `wc`, because the lib/wc config is applied after the vue.config.js.

**Comment by LinusBorg** [comment]
Please open a new issue with a reproduction if you have a problem. This issue is closed and the chances of your information being processed is close to zero.

**Comment by doublemarked** [comment]
@LinusBorg I included a project that is configured to build with `--target lib`, and specific instructions on how to reproduce the issue. Can you help me understand what more would help? The `vue inspect` command does not support `--target lib`

**Comment by Shinigami92** [comment]
I had to switch from `"removeComments": true` to `"removeComments": false` in my `tsconfig.json`
Hope this can save someone hours of wasted time...
